tests/api/openapi/templates/test_fastapi_codegen_template.py

```diff
@@ -15,7 +15,9 @@
 assert TEMPLATE_DIR.exists(), TEMPLATE_DIR
 
 
-@pytest.mark.parametrize("openapi_file_path", OPENAPI_FILE_PATHS)
+@pytest.mark.parametrize(
+    "openapi_file_path", OPENAPI_FILE_PATHS, ids=[p.name for p in OPENAPI_FILE_PATHS]
+)
```

Comment:
add test checking that the original (unpatched) method raises an error. If not, the test should fail.